magda-web-client/src/config.js

```diff
@@ -91,6 +92,14 @@ export const config = {
         {
             category: "Developers",
             links: [["Powered by Magda", "https://github.com/TerriaJS/magda/"]]
+        },
+        {
+            category: "About",
+            links: [
+                ["About data.gov.au", "page/about"],
```

Comment:
Did you mean to have About and Privacy Policy in two different places?

Reply:
Merge accident - we only need one column of the links about the site.